### PR TITLE
copyable embed tag

### PR DESCRIPTION
**Pull request by jessopb**
EmbedArea similar to copyable text.
Needs styling.

### CODE REVIEW

File: src/ui/component/embedArea/view.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        label={label}
+        value={embedText || ''}
+        ref={input}
+        readOnly
+        onFocus={onFocus}
+      />
+      <div className="card__actions card__actions--center">
+        <Button
+          icon={ICONS.COPY}
+          button="inverse"
```
Comment (seanyesmunt) [inline src/ui/component/embedArea/view.jsx]: This can just be `button="link"` to look like the facebook/twitter links above

Comment (jessopb) [reply src/ui/component/embedArea/view.jsx]: done

**Comment by jessopb** [comment]
> This looks good. It doesn't seem to actually copy the link when I click the copy button though.

Ah, fixed that.

**Comment by jessopb** [comment]
Still needs a solution for the best way to give it the proper input box border/focus behavior.

**Pull request merged**